slipcover/bytecode.py

```diff
@@ -43,11 +43,13 @@ def branch2offset(arg: int) -> int:
 op_POP_TOP = dis.opmap["POP_TOP"]
 op_JUMP_FORWARD = dis.opmap["JUMP_FORWARD"]
 op_NOP = dis.opmap["NOP"]
+op_STORE_NAME = dis.opmap["STORE_NAME"]
+op_STORE_GLOBAL = dis.opmap["STORE_GLOBAL"]
 
 
 def arg_ext_needed(arg: int) -> int:
     """Returns the number of EXTENDED_ARGs needed for an argument."""
-    return (arg.bit_length() - 1) // 8
+    return -((arg>>8).bit_length() // -8)   # ceiling by way of -(a // -b) 
```

Comment:
Pretty opaque stuff here - is it performance sensitive?

Reply:
It was the incorrect calculation for some cases.  I added tests and the bug became apparent.
I tried reverting, it's not what caused the 2% slowdown.